### PR TITLE
Updated config.py to address issue #2927

### PR DESCRIPTION
**Pull request by jtranquilli**
delete_after_days was previously hard-coded to be 60, changed it to a much larger value, 700 as requested in the issue.
From what I can tell, the error message is not hard-coded, and the change in value of delete_after_days is now reflected in the error message, i.e. the error would now be updated to: 
'Error: delete_after_days: Number must be between 0 and 700'
If you were to choose a value of greater than 700.

### CODE REVIEW

File: frontend/coprs_frontend/coprs/config.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ class Config(object):
     CACHE_KEY_PREFIX = "copr_cache_"
 
     # Default value for temporary projects
-    DELETE_AFTER_DAYS = 60
+    DELETE_AFTER_DAYS = 700 #Changed from 60 to 700 at request of issue #2927
```
Comment (praiskup) [inline frontend/coprs_frontend/coprs/config.py:151]: Thank you for the patch.  I think you need to fix `forms.py` though - not the default value (I think default 60 days is quite OK).

Comment (jtranquilli) [reply frontend/coprs_frontend/coprs/config.py:151]: Very good catch, thanks for the response. I'll update line 559 of forms.py to reflect the change.

**Comment by praiskup** [comment]
Can we have to limit rounded to say 1000 days, or 2x365 at least? :)

File: frontend/coprs_frontend/coprs/config.py (hunk 1 above)
Comment (praiskup) [reply frontend/coprs_frontend/coprs/config.py:151]: +1, can you drop the first commit and rebase?

Comment (jtranquilli) [reply frontend/coprs_frontend/coprs/config.py]: I set the default value in config.py back to what it was before, and rounded the max value in forms.py  as requested, to be 730 days, which is 2 years. I'll drop the first 2 commits now.

**Comment by jtranquilli** [comment]
@praiskup Sorry, I accidentally force pushed this pull request (?). 
Screenshot 2023-09-26 at 5 12 07 PM

I did implement the changes you asked for though.

**Pull request closed without merging**